### PR TITLE
Update single route definition

### PR DESCRIPTION
**Pull request by rita3ko**

### CODE REVIEW

File: src/content/cli-wrangler/configuration.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Keys to configure per project in your `wrangler.toml`.
   - This is a boolean flag that specifies if your worker will be deployed to your [workers.dev](https://workers.dev) subdomain. If omitted defaults to false.
 
 - `route` <Type>not inherited</Type> <PropMeta>optional</PropMeta>
-  - A list of routes you’d like to use your worker on. These follow exactly the same rules a `route`, but you can specify a list of them.<br />`routes = ["http://example.com/hello", "http://example.com/goodbye"]` One key of `route`OR `routes` is only if you are not using a [workers.dev](https://workers.dev) subdomain.
+  - A route, specificied by URL pattern, on your zone that you would like to run your Worker on. <br />`route = "http://example.com/*"`. One key of `route` OR `routes` is only if you are not using a [workers.dev](https://workers.dev) subdomain.
```
Comment (signalnerve) [inline src/content/cli-wrangler/configuration.md:57]: +1. I do wonder if the `One key of route or routes is only if...` is kinda confusing verbiage but I know it's pre-existing. not a blocker

**Review by signalnerve — APPROVED** [review]

File: src/content/cli-wrangler/configuration.md (hunk 1 above)
Comment (jclee) [reply src/content/cli-wrangler/configuration.md:57]: s/specificied/specified

Comment (jclee) [reply src/content/cli-wrangler/configuration.md:57]: Also, it feels like there is a word missing from "One key of `route`OR `routes` is only if..."

**Comment by rita3ko** [comment]
closing this in favor of https://github.com/cloudflare/cloudflare-docs/pull/40

**Pull request closed without merging**